docs/release-notes/8.11.asciidoc

```diff
@@ -1,10 +1,34 @@
 [[release-notes-header-8.11.0]]
 == 8.11
 
+[discrete]
+[[release-notes-8.11.4]]
+=== 8.11.4
+
+[discrete]
+[[known-issue-8.11.4]]
+==== Known issues
+
+* When users filter indicator data from the Intelligence page, the KQL bar shows the correct field name, but the field value is `Warning`. 
```

Comment:
```suggestion
* When you filter indicator data from the Intelligence page, the KQL bar shows the correct field name, but the field value is `Warning`. 
```
Nbd, but figured I'd suggest this since we usually use 'you' in the release notes

Reply:
Removing this as per the comment [here](https://github.com/elastic/security-docs/issues/4582#issuecomment-1885707682).

Reply:
Resolved in [73df9b7](https://github.com/elastic/security-docs/pull/4547/commits/73df9b78592559370733def316fac81024104b19)